services/121-service/package.json

```diff
@@ -12,9 +12,10 @@
   "scripts": {
     "lint": "eslint \"**/*.{ts,js}\"",
     "fix": "npm run lint -- --fix",
-    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION)  node -r tsconfig-paths/register index.js",
+    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION)  nest start -r tsconfig-paths/register -p tsconfig.json",
+    "start:dev": "nodemon --watch src/**/*.json --exec npm run start:dev:nest",
```

Comment:
`resolveJsonModule` Does not work the same for swc as for tsc. The result was that you have to restart docker to apply changes you made to the .json files in de seed-data. I therefore added nodemon so it also recompiles on a .json file change